index.html

```diff
@@ -21,18 +22,29 @@
       integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4="
       crossorigin="anonymous"
     ></script>
-    <link href="index.css" rel="stylesheet" />
+    <script src="menuscroll.js"></script>
+    <script src="menuchange.js"></script>
 
+    <link href="index.css" rel="stylesheet" />
     <title>UNBAP</title>
   </head>
+
   <body>
+    <div class="menuBar">
+      <span onclick="scrollToPart1()">&nbsp;&nbsp;Home&nbsp;&nbsp;</span>|<span
+        onclick="scrollToPart2()"
+        >&nbsp;&nbsp;Members&nbsp;&nbsp;</span
+      >|<span onclick="scrollToPart3()">&nbsp;&nbsp;Projects&nbsp;&nbsp;</span>
```

Comment:
`&nbsp` 를 사용하는 것을 그렇게 좋은 습관이 안닙니다. margin 또는 padding 을 이용해보세요.

```suggestion
      <span class="menu" onclick="scrollToPart1()">Home</span>|<span
        class="menu"
        onclick="scrollToPart2()"
        >Members</span
      >|<span class="menu" onclick="scrollToPart3()">Projects</span>
```
이런 방식으로 class 를 사용한 다음, css margin 을 지정해주세요.

```css
.menuBar .menu {
  margin: auto 1em;
}
```